docs/commands/cp.md

```diff
@@ -0,0 +1,155 @@
+# cp
+
+Copy storage files and directories between cloud and local storage.
+
+## Synopsis
+
+```usage
+usage: datachain cp [-h] [-v] [-q] [-r] [--team TEAM]
+    [-s] [--anon] [--update] [--no-glob]
+    [--force] source_path destination_path
+```
+
+## Description
+
+This command copies files and directories between local and/or remote storage. This uses the credentials in your system by default or can use the cloud authentication from Studio.
+
+## Arguments
+
+* `source_path` - Path to the source file or directory to copy
+* `destination_path` - Path to the destination file or directory to copy to
+
+## Options
+
+* `-r`, `-R`, `--recursive` - Copy directories recursively
+* `--team TEAM` - Team name to copy storage contents to
+* `-s`, `--studio-cloud-auth` - Use credentials from Studio for cloud operations (Default: False)
+* `--anon` - Use anonymous access to storage
+* `--update` - Update cached list of files for the source for local approach
+* `--no-glob` - Do not expand globs (such as * or ?) for local approach
+* `--force` - Force creating files even if they already exist
+* `-h`, `--help` - Show the help message and exit
+* `-v`, `--verbose` - Be verbose
+* `-q`, `--quiet` - Be quiet
+
+## Copy Operations
+
+The command supports two main modes of operation:
+
+### Default Mode
+By default, the command operates directly with local storage access, supporting various copy scenarios between local and remote storage.
```

Comment:
`local storage access` this is very confusing, considering that we have already things like "_local_ and remote storage"

Reply:
Updated.